Templates/JenkinsPipeline/README.md

```diff
@@ -0,0 +1,132 @@
+# Jenkins Pipeline template
+
+This template provides an Jenkinsfile [Jenkinsfile](Jenkinsfile) definition to setup a Jenkins Multibranch pipeline for applications managed in any Git provider.
+
+## Overview and capabilities
+
+This pipeline template is implementing the [Git-based process and branching model for mainframe development](https://ibm.github.io/z-devops-acceleration-program/docs/git-branching-model-for-mainframe-dev) with Jenkins as a the Pipeline Orchestrator.
+
+It leverages the [Common Backend scripts](https://github.com/IBM/dbb/blob/main/Templates/Common-Backend-Scripts/README.md) in the Build, Packaging and Deployment stages.
+
+The pipeline implements the following stages
+
+* `Checkout` stage to the Git repository to a workspace directory on z/OS Unix System Services using the integrated Git Plugin of Jenkins.
+* `Pipeline Setup` computing the settings for subsequent stages, that are displayed in the `Parameters and Values` stage.
+* `Build` stage 
+  * to invoke zAppBuild via the [dbbBuild.sh](../Common-Backend-Scripts/README.md#42---dbbbuildsh) Common Backend script,
+  * to publish log files to the Jenkins build result.
+* `SonarQube Analysis` stage to request a SonarQube scan of the application repository.
+* `Packaging` stage
+  * to create a UrbanCode Component version via the Common Backend script [ucdPackaging.sh](../Common-Backend-Scripts/README.md#45---ucdpackagingsh),
+  * to publish packaging log files to the Jenkins build result,
+  * to add links to the UCD Component version.
+* `Deploy to INT` stage to deploy to the development / integration test environment that includes:
+  * to request the Deployment of the UrbanCode Component version via the Common Backend script [ucdDeploy.sh](../Common-Backend-Scripts/README.md#46---ucddeploysh) to the shared Development/Integration Test environment,
+  * to publish deployment log files to the Jenkins build result,
+  * to add links to the UCD Deployment request.
+ * `Workspace Cleanup` stage to clean up the workspace
+
+Depending on your selected and software analysis and deployment technology, review the definitions and (de-)/activate the appropriate steps.
+
+The pipeline is implemented as a declarative pipeline.
+
+## Prerequisites
+
+The [Common Backend scripts](../Common-Backend-Scripts/) need to be configured for the selected technologies to operate correctly.
+
+## Installation and setup of template
+
+**Note: Please work with your pipeline specialist to review the below section.**
+
+The `Jenkinsfile` can be dropped into the root folder of your Git repository and will automatically provide pipelines for the specified triggers defined in the Jenkins Multibranch Pipeline job configuration.
```

Comment:
I'd say 'placed' instead of 'dropped', bu that's fine.
Or a rephrasing like:
When placed into the root folder of your Git repository, the `Jenkinsfile` automatically provides pipelines for the specified triggers defined in the Jenkins Multibranch Pipeline job configuration.